tests/fixtures/issues/issue-900.php

```diff
@@ -46620,19 +46620,19 @@
             'JavaApplets' => false,
             'ActiveXControls' => false,
             'isMobileDevice' => true,
-            'isTablet' => false,
+            'isTablet' => true,
             'isSyndicationReader' => false,
             'Crawler' => false,
             'isFake' => false,
             'isAnonymized' => false,
             'isModified' => false,
             'CssVersion' => '3',
             'AolVersion' => '0',
-            'Device_Name' => 'general Mobile Phone',
```

Comment:
This is that useragent that should be a tablet, but the device wasn't defined on the pattern.